orttraining/orttraining/eager/ort_aten.cpp

```diff
@@ -503,48 +503,40 @@ void resize_impl_ort_(
 
 namespace aten {
 
-at::Tensor empty_memory_format(
+at::Tensor empty_strided(
   at::IntArrayRef size,
-  // *,
+  at::IntArrayRef stride,
   c10::optional<at::ScalarType> dtype_opt,
-  c10::optional<at::Layout> layout_opt,
-  c10::optional<at::Device> device_opt,
-  c10::optional<bool> pin_memory,
-  c10::optional<at::MemoryFormat> memory_format) {
-  ORT_LOG_FN(size, dtype_opt, layout_opt, device_opt, pin_memory, memory_format);
-
-  assert(dtype_opt.has_value());
-  assert(device_opt.has_value());
+  c10::optional<at::Layout> layout_opt,  // Ignored because there's no ONNX support.
```

Comment:
Before there was a TODO stmt for some of these, are we now saying that it is ok to silently ignore these?

Reply:
yes, based on symbolic opset behavior